src/components/inputs/Label/Label.jsx

```diff
@@ -28,7 +30,7 @@ export default function Label({
   return (
     // disabling label-has-associated-control as eslint does not see role attribute as a nested control
     // eslint-disable-next-line jsx-a11y/label-has-associated-control
-    <label className={cssClasses} style={style}>
+    <label className={cssClasses} style={style} id={ID}>
```

Comment:
id={labelId}

the idea is that this id is intented just for the label, not for the whole component. Currently the label is the whole component, but we will change that in the future to:

```jsx
<div className={cssClasses} style={style}>
  <label className={sc('label-text'}} id={labelId} />
  <div className={sc('control')} />
  <div className={sc('error')} />
</div>